regolith/resolver.go

```diff
@@ -52,123 +51,155 @@ func resolveResolverUrl(url string) (string, error) {
 	return fmt.Sprintf("git::https://%s/%s", repoUrl, path), nil
 }
 
-// DownloadResolverMaps downloads the resolver.json files
-func DownloadResolverMaps() error {
+// resolveResolverUrl resolves the resolver URL from the short name to a full
+// URL that can be used by go-getter
+func resolveResolverUrlNew(url string) (string, string, error) {
+	urlParts := strings.Split(url, "/")
+	if len(urlParts) < 4 {
+		return "", "", burrito.WrappedErrorf(
+			"Incorrect URL format.\n" +
+				"Expected format:" +
+				"github.com/<user-name>/<repo-name>/<path-to-the-resolver-file>")
+	}
+	repoUrl := strings.Join(urlParts[0:3], "/")
+	path := strings.Join(urlParts[3:], "/")
+	return fmt.Sprintf("https://%s", repoUrl), path, nil
+}
+
+// DownloadResolverMaps downloads the resolver repositories and returns lists of urls and paths
+func DownloadResolverMaps(forceUpdate bool) ([]string, []string, error) {
 	Logger.Info("Downloading resolvers")
```

Comment:
This message probably shouldn't print when the download is on the cooldown.

Reply:
This message is now removed. I can't do just one message when it's out of cooldown, because each repo has a separate cooldown. I added messages about updating/downloading resolvers instead.